Cooper/ClientApp/src.v2/app/layouts/profile/profile.layout.html

```diff
@@ -1,3 +1,38 @@
-<p>
-  profile works!
-</p>
+<div>
+  <main>
+      <div class="col-md-1"></div>
+
+      <div class="col-md-2">          
+          <coop-my-profile
+          *ngIf = "isSessionProfile"
```

Comment:
unclear name

Reply:
coop-my-profile or name of bolean variable ?